From 7259d82321814393bd59f667623130e44b6ed26b Mon Sep 17 00:00:00 2001
From: K. Y. Srinivasan <kys@microsoft.com>
Date: Mon, 6 Jun 2011 15:49:46 -0700
Patch-mainline: 3.1-4c4
Subject: [PATCH] Staging: hv: vmbus: Get rid of the call to dump channel state in channel event handler

Get rid of the call to dump channel state in channel event handler.

Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Haiyang Zhang <haiyangz@microsoft.com>
Signed-off-by: Abhishek Kane <v-abkane@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/channel.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/hv/channel.c b/drivers/staging/hv/channel.c
index a4bee81..7854de2 100644
--- a/drivers/staging/hv/channel.c
+++ b/drivers/staging/hv/channel.c
@@ -885,7 +885,6 @@ EXPORT_SYMBOL_GPL(vmbus_recvpacket_raw);
  */
 void vmbus_onchannel_event(struct vmbus_channel *channel)
 {
-	dump_vmbus_channel(channel);
 
 	channel->onchannel_callback(channel->channel_callback_context);
 
-- 
1.6.0.2

